src/TLuaInterpreter.cpp

```diff
@@ -10357,24 +10357,20 @@ int TLuaInterpreter::registerAnonymousEventHandler(lua_State* L)
 }
 
 
-int TLuaInterpreter::Send(lua_State* L)
+int TLuaInterpreter::expandAlias(lua_State *L)
 {
     string luaSendText;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "Send: wrong argument type");
+        lua_pushstring(L, "expandAlias: wrong argument type");
```

Comment:
Oh, can we have the error message updated completely?

Reply:
I'm just looking to fix https://github.com/Mudlet/Mudlet/issues/582 here

Reply:
OK - it'll get cleaned up _sometime_...